DTML.EduBot/Dialogs/GreetingDialog.cs

```diff
@@ -9,13 +9,14 @@
 
 namespace DTML.EduBot.Dialogs
 {
-    public partial class ChitChatDialog : QnaLuisDialog<object>
+    public partial class NoneDialog : QnaLuisDialog<object>
```

Comment:
The class name should not be changed to NoneDialog. Right now ChitChatDialog reflects the fact the bot will try its best to respond.

Plus ChitChatDialog is a partial class that is shared for all different intents.